assets/js/modules/pagespeed-insights/components/common/LabReportMetrics.js

```diff
@@ -128,10 +128,10 @@ export default function LabReportMetrics( { data, error } ) {
 				</tbody>
 			</table>
 			<div className="googlesitekit-pagespeed-report__row googlesitekit-pagespeed-report__row--last">
-				<ReportDetailsLink />
-				<Button onClick={ updateReport }>
+				<Link href="#update-pagespeed" onClick={ updateReport }>
```

Comment:
The `href` shouldn't be needed here. It should render as a `button` that looks like a link without it. Can we avoid adding this?

Reply:
Oh, sweet, done!